charms/reactive/relations.py

```diff
@@ -474,6 +474,7 @@ def set_state(self, state, scope=None):
         See :meth:`conversation` and :meth:`Conversation.set_state`.
         """
         self.conversation(scope).set_state(state)
+    set_flag = set_state
```

Comment:
The formatting looks really weird on this file; is this mixed tabs and spaces?

Reply:
Ah, I missed that these are still in here.  They're actually method aliases, but they should have been cleaned up with the rest of the streamlining of the PR.

Reply:
Oops, these changes come from #233.  I'm not sure why they're showing up in this PR.

Reply:
Rebased to resolve.